python/servo/devenv_commands.py

```diff
@@ -34,6 +34,17 @@ def cargo(self, params):
         if not params:
             params = []
 
+        cargo_paths = [path.join('components', 'servo'),
```

Comment:
These are also used in `update_cargo`, so this can be moved to the top level as a constant. Also, formatting nit, put each path on a new line with a trailing comma:
```
CARGO_PATHS = [
    path.join(...),
]